src/Hl7.Fhir.Support/Specification/ElementDefinitionSummary.cs

```diff
@@ -16,14 +16,15 @@ public class ElementDefinitionSummary : IElementDefinitionSummary
         private ElementDefinitionSummary() { }
 
         public ElementDefinitionSummary(string elementName, bool isCollection, bool isChoice,
```

Comment:
This is a public constructor on a public class. We cannot change this because we are going to affect the interface. Better is to create an overload of this constructor with an extra parameter

Reply:
Added new overload of constructor